mods/ts/sequences/misc.yaml

```diff
@@ -1,15 +1,11 @@
 overlay:
 	Defaults:
-		Start: 1
 		Offset: 0, -12
 	build-valid-snow: place
-		Start: 0
 	build-valid-temperat: place
-		Start: 0
 	build-invalid: place
+		Start: 1
 	target-select: place
-	target-valid-desert: place
-	target-valid-interior: place
 	target-valid-snow: place
```

Comment:
Turns out that target-select is the only version of these that is used in TS. The other ones are only used by the RA chronoshift power.